File: FrontEnd/src/app/models/cart/cartLine.ts
import { Colour } from '../products/colour';
import { Product } from '../products/product';
import { Size } from '../products/size';
import { Cart } from './cart';

export interface CartLine {
  cartLineId: number;
  cartQuantity: number;
  cart: Cart;
  product: Product;
  colour: Colour;
  size: Size;
}

File: SheCollectionBE/SheCollectionBE/DB Models/CartLine.cs
using System;
using System.Collections.Generic;

namespace SheCollectionBE.Models
{
    public partial class CartLine
    {
        public int CartLineId { get; set; }
        public int CartQuantity { get; set; }

        public virtual Cart Cart { get; set; }
        public virtual Product Product { get; set; }
        public virtual Colour Colour { get; set; }
        public virtual Size Size { get; set; }
    }
}
